src/components/AttachmentModal.js

```diff
@@ -277,8 +277,9 @@ class AttachmentModal extends PureComponent {
                     {this.props.onConfirm && (
                         <Animated.View style={StyleUtils.fade(this.state.confirmButtonFadeAnimation)}>
                             <Button
+                                key={this.props.isSmallScreenWidth}
```

Comment:
`key` is needed to re-render button when screenWidth changes. This way the button size will change if user decreases/increases the screen size at web/desktop

Reply:
I think it should work without the key prop.

Reply:
Found the problem. The styles are in the constructor that's why it is not working correctly. https://github.com/Expensify/App/blob/5c5008b2dfa321359aa962f756b05c0c914c71c2/src/components/Button.js#L147
@Puneet-here, please remove `this.additionalStyles` and instead pass `...StyleUtils.parseStyleAsArray(this.props.style) `https://github.com/Expensify/App/blob/5c5008b2dfa321359aa962f756b05c0c914c71c2/src/components/Button.js#L259

Reply:
Updated!